jboss-set/zeus
# SET-420 setup molecule test for role motd

Issue: https://issues.redhat.com/browse/SET-420

To test it you need to install podman, python and molecule in your local system.
Here in this molecule setup it is testing the below scenarios:
1. checking syntax of the `motd` role playbooks.
2. 
-     In **create** action, using podman it'll create an image for rhel 8 and then create an instance from that image.
-     In **converge** action, it'll run motd playbooks in the instance to check if the role is running fine or not.
-     In **verify** action, it'll run verify.yml against the instance, to check the motd output is fine or not. Which will save us from manually checking the motd output.
-     In **destroy** action, it'll remove all the instances.

How to test:

Run `molecule test` it's the easiest way.
Otherwise run `molecule converge`, `molecule verify`, ` molecule destroy` in a sequence.
